Auth0/Optional+DebugDescription.swift

```diff
@@ -0,0 +1,32 @@
+// Optional+DebugDescription.swift
```

Comment:
Is putting `+` in the filename a common thing to do? Where would one normally do that? Same question for the other files named in such a way

Reply:
In ObjC when you declared an extension you had to give it a name. The convention was to include that name in the extension filename, like this: **class being extended + extension name**.  In Swift you don't have to give names to extension blocks, but the convention remains.